src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = 40;
+  const numberOfDays = days;
+  const DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (numberOfDays < 3) {
+    return numberOfDays * basePrice;
+  }
+
+  if (numberOfDays >= 7) {
```

Comment:
Consider defining a constant for the minimum days required for a long term discount to make the code more expressive and align with the checklist.